Source/Client/BuildResults.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Dolittle.Runtime.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Dolittle.Runtime.Client;
+
+/// <summary>
+/// Represents a <see cref="ReadOnlyCollection{T}"/> of <see cref="BuildResult"/>.
+/// </summary>
+public class BuildResults
+{
+    public static BuildResults Empty => new(
```

Comment:
This will create it every time. If it's immutable, just do it once?

Reply:
Yep. Changed